recipes/r-rserve/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
This one needs a special build.sh, because it directly links against some R stuff. You find the necessary modifications in conda/conda-recipes. This will solve your libRblas.so error.